### ISSUE TITLE
Facette - Verlag

### ISSUE THREAD

**Issue by thefass** [ubtue/tuefind#1672]
Die Verlagsfacette hat 61,5% unzugewiesene.
Dieser hohe Anteil rührt daher, dass in Aufsätzen und Rezensionen kein Verlag angegeben wird.
Der Verlag steht nur in der Gesamtaufnahme.

Beispiel: https://www.ixtheo.de/Record/389463434#detail
![Bildschirmfoto 2021-09-21 um 21 30 58](https://user-images.githubusercontent.com/14169098/134235587-524f3cdb-1b7d-44d7-bfc4-367d7da7241d.png)

Der Verlag steht in 264 $b

Allerdings sind die Aufsätze und Rezensionen über die PPN mit der Gesamtaufnahme verknüpft. Über diese Verknüpfung kann man die Verlagsangabe aus der Gesamtaufnahme übernehmen.
PPN der Gesamtaufnahme: 389463434
PPN im unselbständigen Titel:
![Bildschirmfoto 2021-09-21 um 21 42 17](https://user-images.githubusercontent.com/14169098/134237046-a8c3e3b6-634b-4de6-99e0-23c4f3ba5309.png)

Aufgabe: bitte in der Pipeline die Verlagsangabe in den Aufsatztitel / Rezension nachführen.
Achtung: es können auch die früheren Verlage in 264 $b angegeben sein. Deswegen muss der aktuellste genommen werden.

**Comment by mtrojan-ub** [comment]
Die Informationen sind ja bereits in 773d vorhanden. Anstatt die Informationen vom übergeordneten Werk zum Artikel zu kopieren, werten wir jetzt zusätzlich die Information aus 773 aus, siehe ptah.

Zusätzlich haben wir eine Anpassung vorgenommen, die die alphabetische Sortierung verbessert (diverse Anführungszeichen werden entfernt).

Außerdem ist nun ein weiterer Teil des Algorithmus aktiv, der bisher aufgrund eines Bugs nicht funktioniert hat. In manchen Fällen steht als Information z.B. nur "Madrid, 1961" ohne Verlag zur Verfügung. Diese Fälle werden aktuell mit "[Publisher not identified]" markiert. In vielen Fällen würde es hier nichts bringen die Information vom übergeordneten Werk zu übernehmen, da beim übergeordneten Werk die Information ebenfalls nicht zur Verfügung steht.

Beispiel: https://ptah.ub.uni-tuebingen.de/Record/176803303X

**Comment by thefass** [comment]
ok, das sieht gut aus.
Eine Nachfrage: was ist der Unterschied zwischen [publisher not identified] und [publication not identified] ?

**Comment by thefass** [comment]
Manche Verlage kommen mit unterschiedlichen Schreibweisen vor. Diese müßten normalisiert und kumuliert werden. Wir machen so etwas auch schon in anderen Bereichen. M.W. gibt es eine Liste auf GitHub, wo wir so etwas eintragen können. Könnte @jriedl wissen.
Bitte auch für die Verlage einrichten.

Beispiel:
Vandenhoeck & Ruprecht
Vandenhoeck und Ruprecht

**Comment by relhei** [comment]
@thefass: Wollen wir diese Anzeigen [publisher not identified] und [publication not identified] tatsächlich so stehen lassen? Sie werden in jedem Titelsatz, in dem die entsprechenden Felder nicht belegt sind, angezeigt, also auch bei den Nachlass-Titeln, den Alert-Paketen etc.

Bsp.
https://134.2.67.9/Search/Results?lookfor=nachlass&type=AllFields&filter%5B%5D=%7Eformat%3A%22Literary+Remains%22

https://134.2.67.9/Search/Results?type=AllFields&filter%5B0%5D=%7Eformat%3A%22Subscription%20Bundle%22&lng=de

**Comment by mtrojan-ub** [comment]
@thefass kannst Du dazu bitte noch eine Aussage treffen?

**Comment by thefass** [comment]
Ich schalge vor, das so stehen zu lassen. Die Alternative wäre ja, dass in der Facette Verlag der Wert "unzugewiesen" stünde.
Da ist  [publisher not identified] und [publication not identified] genauer.
@relhei Einverstanden?

**Comment by relhei** [comment]
Zugegebenermaßen verstehe ich den inhaltlichen Unterschied zwischen [Verlag nicht identifiziert] und [Publikation nicht identifiziert] immer noch nicht ganz. Klar ist, dass die unterschiedlichen Formulierungen mit der unterschiedlichen (Nicht-)Belegung der Felder zu tun haben. Aber im Endeffekt fehlt doch in beiden Fällen die Verlagsangabe (bzw. Verlag **und** Verlagsort). Verwirrt es Nutzer*innen nicht, wenn dann "publication not identified" steht? Eigentlich ist doch mit "publication" der Ort der Veröffentlichung "in Tateinheit" mit dem Verlag gemeint, oder liege ich falsch? Den Publikationsort erfassen wir aktuell nicht in einer gesonderten Facette. Und kommt [publication not identified] überhaupt jemals ohne die Angabe [publisher not identified] im selben Titel vor? Wäre es dann nicht ausreichend, nur "publisher not identified" anzugeben und in die Verlags-Facette zu übernehmen?

Überlegenswert wäre noch, diese Hilfstexte zu übersetzen. Lässt sich das einfach umsetzen?
Vorschlag für Deutsch: [Verlag nicht identifiziert] und, wenn wir dabei bleiben, [Publikation nicht identifiziert].

**Comment by mtrojan-ub** [comment]
@relhei => Definition siehe Eintrag vom 8.10.21. Wie müsste diese Definition denn genau geändert werden, damit es Deinem Verständnis entspricht?

**Comment by andreas-ub** [comment]
Hier noch ein Beispieldatensatz, bei den beide Flags gesetzt sind:
Ptah-Relbib: ID 1797405748

Published:    [publisher not identified] 2022
[publication not identified]

264         1     |c 2022
260 b <nicht vorhanden>
773     0     8     |i Enthalten in  |t British Journal of religious education  |d London, 1978 <nicht vollständig>

**Comment by relhei** [comment]
Zitat aus meinem Kommentar vom 21.3.:
"Und kommt [publication not identified] überhaupt jemals ohne die Angabe [publisher not identified] im selben Titel vor?"
Gibt es darauf eine Antwort?

**Comment by andreas-ub** [comment]
in den MARC-Daten scheint das Feld 260b nicht mehr befüllt zu sein.
Daher sind vermutlich alle [publisher not identified] auch [publication not identified], da 773d (falls korrekt ausgefüllt) beide Flags entfernt und der publisher momentan nicht mehr aus 260b kommt.

Wenn nur ein Feld [publisher not identified] verwendet wird, wäre die Klassifikation diser beiden Datensätze identisch:
https://ptah.ub.uni-tuebingen.de/Record/1106517911#details
264 |a Romae  |b [Cunial]  |c 1969   :  hier ist publication vorhanden (zusätzlich: publisher nicht vorhanden)
wäre dann gleichwertig mit z.B.:
https://ptah.ub.uni-tuebingen.de/Record/1137976586#details
264 |a Milano  |c 19XX-  :  hier ist publication nicht vorhanden (zusätzlich: publisher nicht vorhanden)

Allerdings variieren die 264-Unterfelder in ihrer Genauigkeit, die Prüfung auf "...not identified" bezog sich vermutlich auf die Vollständigkeit innerhalb der Felder.

**Comment by andreas-ub** [comment]
Die Änderungen sind nun auf ptah aktiv -> ready for testing.
Z.B. die beiden Einträge von oben (https://ptah.ub.uni-tuebingen.de/Record/1106517911#details und https://ptah.ub.uni-tuebingen.de/Record/1137976586#details) werden nun beide als  [publisher not identified] gelistet.

**Comment by thefass** [comment]
ok, Danke

**Comment by jriedl** [comment]
Wie im FID-Meeting besprochen, wird [Publisher not identified] jetzt nicht mehr vergeben, wenn die Information bereits aus einer anderen Quelle (z.B. 773) bestimmt werden kann. Aktueller Stand auf ptah.

**Comment by socheres** [comment]
bitte auch auf ixtheo nachziehen.

**Comment by thefass** [comment]
Führen wir irgendwo eine Konkordanz, so dass wir z.B. die Treffer unter "Oxford Univ. Press" und unter "Oxford University Press" zusammenführen können?

**Comment by jriedl** [comment]
> bitte auch auf ixtheo nachziehen.

Sollte nach Abschluss des nächsten Imports, voraussichtlich morgen früh, sichtbar sein.

**Comment by jriedl** [comment]
> Führen wir irgendwo eine Konkordanz, so dass wir z.B. die Treffer unter "Oxford Univ. Press" und unter "Oxford University Press" zusammenführen können?

Ja: https://github.com/ubtue/ub_tools/blob/master/cpp/data/publisher_normalisation.map, da ist das OUP-Mapping sogar bereits vorhanden. Allerdings ist es derzeit auf 260b und 264b beschränkt, die Aufteilung innerhalb der Facette kommt aber wohl aus Werten in 773d, worin noch weitere Informationen enthalten sind. Das entsprechende Programm und das Pipelinescript wurde nun angepasst, Ergebnis ist nach Abschluss des nächsten Imports, voraussichtlich morgen früh, auf ptah sichtbar.

**Comment by thefass** [comment]
geklärt. Das PÜroblem der Normalisation besteht m.E. noch, dafür eröffne ich aber ein eigenes Issue

**Comment by IxTheoKm** [comment]
Die Einträge sehen jetzt wieder so aus:

![Image](https://github.com/user-attachments/assets/37ce0d35-140d-4ab2-995f-ef9c05bb9c67)
